### DISCUSSION TITLE
Multiple CANs for same procurement with multiple lots

### DISCUSSION THREAD

**Discussion by DeanFirkelj** [OP-TED/eForms-SDK#556] [Q&A] [answered]
Hi,
In case when a procurement contains multiple lots (for example lots A, B & C) and we have following scenario
- A is awarded and contracted
- B is still not known i.e. review process is still in progress
- C is cancelled

The first notice (CAN 1) should include info about A & C (no B, since there is no final result).

At the stage when B is "finished", let's say it is awarded after the review process, should the new CAN (CAN 2), contain information about:
I) Only B or
II) A, B & C, although A & C are repetition of what was presented in CAN 1

If II) is correct, what would happen if A & C have different values in CAN 2 than in CAN 1? Will the validation fail, or not?
I'm aware (in principle) that Change notice referring to CAN 1 should be used for "errors" in CAN 1.

If I) is correct, is it still possible to transmit A, B & C like in scenario II)?

Thanks!

**Comment by YvesJo** [comment, marked as answer]
Hi,
Following the scenario, you should publish the first notice with the following information:

- Lot A has been awarded and the contract signed,
- competition for Lot B has not been awarded and competition is ongoing
- Lot C has not been awarded and competition is closed in addition to the non-award reason

The second result notice should only contain the result of the Lot B award process

If for any reason, the information about Lot A has to change (i.e. in case of a corrigendum), then you'll have to publish a separate result notice pointing to the first notice and containing all the information required for a change.
